CMakeLists.txt

```
@@ -39,7 +39,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # Python and Cython requirements
 set(PYTHON_MIN_VERSION 3.9)
-set(CYTHON_MIN_VERSION 0.29.31)
+set(CYTHON_MIN_VERSION 3.0.10)
 
```

```suggestion
set(CYTHON_MIN_VERSION 3.0.10)
set(CYTHON_MIN_VERSION_FALLBACK 0.29.32)
set(CYTHON_MAX_VERSION_FALLBACK 3.0.7)

```

I recommend you also add a `CYTHON_MIN_VERSION_FALLBACK` and `CYTHON_MAX_VERSION_FALLBACK` parameters.

---

On second thought, the max fallback version is not necessary I think.

---

Updated